storage/range_test.go

```diff
@@ -1807,7 +1808,8 @@ func TestInternalPushTxnHeartbeatTimeout(t *testing.T) {
 		// Now, attempt to push the transaction with clock set to "currentTime".
 		tc.manualClock.Set(test.currentTime)
 		args, reply := pushTxnArgs(pusher, pushee, test.pushType, 1, tc.store.StoreID())
-		args.Timestamp = tc.clock.Now()
+		args.Timestamp = proto.Timestamp{WallTime: test.currentTime} // avoid logical ticks
```

Comment:
background processes will add up logical ticks and so the comparisons are random, failing the borderline cases.